boskos/janitor/janitor.go

```diff
@@ -26,15 +28,39 @@ import (
 )
 
 var (
-	clean      = janitorClean
-	POOLSIZE   = 10 // Maximum concurrent janitor goroutines
-	BUFFERSIZE = 1  // Maximum holding resources
+	clean          = janitorClean
+	POOLSIZE       = 10 // Maximum concurrent janitor goroutines
+	BUFFERSIZE     = 1  // Maximum holding resources
+	serviceAccount = flag.String("service-account", "/etc/service-account/service-account.json", "Path to projects service account")
 )
 
+// exists returns whether the given file or directory exists or not
+func exists(path string) (bool, error) {
+	_, err := os.Stat(path)
+	if err == nil {
+		return true, nil
+	}
+	if os.IsNotExist(err) {
+		return false, nil
+	}
+	return true, err
+}
+
 func main() {
 	logrus.SetFormatter(&logrus.JSONFormatter{})
 	boskos := client.NewClient("Janitor", "http://boskos")
 	logrus.Info("Initialized boskos client!")
+
+	// Activate service account
+	if *serviceAccount == "" {
+		logrus.Fatal("--service-account cannot be empty!")
+	}
+
+	cmd := exec.Command("gcloud", "auth", "activate-service-account", "--key-file="+*serviceAccount)
+	if err := cmd.Run(); err != nil {
```

Comment:
Grab the output using something like `cmd.CombinedOutput()` and add it to the log. The error alone might be pretty useless.